ccpm/scripts/pm/epic-merge/run-tests.sh

```
@@ -0,0 +1,31 @@
+# Run tests based on project type
+
+# Look for test commands based on project type
+if [ -f package.json ]; then
+  npm test || echo "⚠️ Tests failed. Continue anyway? (yes/no)"
+elif [ -f pom.xml ]; then
+  mvn test || echo "⚠️ Tests failed. Continue anyway? (yes/no)"
+elif [ -f build.gradle ] || [ -f build.gradle.kts ]; then
+  ./gradlew test || echo "⚠️ Tests failed. Continue anyway? (yes/no)"
+elif [ -f composer.json ]; then
+  ./vendor/bin/phpunit || echo "⚠️ Tests failed. Continue anyway? (yes/no)"
+elif [ -f *.sln ] || [ -f *.csproj ]; then
```

Shellcheck (0.11.0)

[error] -f doesn't work with globs. Use a for loop.

(SC2144)